media/css/mozorg/contribute.scss

```diff
@@ -98,9 +98,58 @@ $image-path: '/media/protocol/img';
             flex-wrap: wrap;
         }
 
+        // protocol override to avoid over-frequent word breaks
+        &.mzp-l-card-quarter .mzp-c-card {
+            @media #{$mq-lg} {
+                @include bidi((
+                    (float, left, right),
+                    (margin-left, 0, $spacing-xl),
+                    (margin-right, $spacing-xl, 0),
+                ));
+                width: calc(50% - (#{$spacing-xl} * 0.5));
+
+                &.mzp-c-card-extra-small,
+                &,
```

Comment:
**Question** what does the single `&` do? I haven't seen that used by itself before

Reply:
`&` has a few capabilities but it all falls under Sass's [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector) behaviour. In this case, it refers to `.contribute-card-container.mzp-l-card-quarter.mzp-c-card`

Eventually, it will probably be [syntactic sugar](https://www.w3.org/TR/css-nesting-1/#nest-selector) for CSS's `@nest` behaviour